INSTALL.md

```diff
@@ -0,0 +1,113 @@
+
+# Installation
+
+k2 supports CUDA as well as CPU. To get the full benefit from k2,
+we recommend that you have installed CUDA on your system before
```

Comment:
Will k2 even run without the CUDA toolkit being installed?

Reply:
Yes, I have tried it locally on a virtual machine without GPU and without CUDA toolkit.
All tests get passed.

The prebuilt wheel package is downloaded from GitHub actions.